LICENSE

```diff
@@ -1,21 +1,117 @@
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+    License Terms and Conditions
+
+0. Definitions
+
+  "This License" refers to the text of the document.
+
+  "The source", "The source code", "the code", "this file" and "these files" are terms which 
+  refer to the files and the code they contain
+  which is covered by copyright and licensed under this license.
+
+  "You" refers to the licensee.
+  The licensee may be either an individual or an organization.
+
+  "Personal use" is a term that applies to both individuals and organizations.
+  It is defined as use which is solely by and solely for an individual
+  and not for the purposes of profit or redistribution.
+
+  "Contribution" refers to any code given to the primary 
+  license holder for use by the license holder to improve
+  or otherwise modify the prior state of the code.
+
+  "Fork" refers to a fork of the original source code as defined
+  by the host provider's terms.
+
+  "Pull request" refers to an action which is intended to allow 
+  third-party contributions to the original licensed code,
+  by the means provided by the host provider's terms.
+
+1. Usage
+
+  You are permitted under this license to
+  download, copy, use, and compile this code for personal use.
+
+  Your personal use may extend to users of other applications this code is used in
+  so long as you do not distribute this code or otherwise imply a license to
+  additional individuals or organizations.
+
+  Under this license, the code provided may not be used in relation with any of the following
+  categories as the code has not been deemed suitable, and the owner would like to ensure
+  the code can not be used in connection with specific high-risk applications without being
+  further evaluated for safety.
+
+	- Operating a medical facility.
+	- Operating a facility which deals with nuclear fusion or nuclear fission.
+	- Operating a facility which deals with the manufacture of weapons.
+	- Operating a facility which deals with the manufacture of medicine.
+	- Handling information which is protected by any government secrecy laws.
+	- Handling information which is protected by any medical privacy laws.
+
+2. Modifications
+  
+  You are free to make modifications for personal use as covered under section 1 (Usage)
+
+  You may also make modifications for the additional purpose of contributions to the
+  original source code.
+
+3. Redistribution
+
+  You may not redistribute the code or modifications made to it
+  beyond the purpose of making a pull request to the original project.
+
+  At the time which a pull request is closed for any reason,
+  you must cease any distribution of the modification within 14 days of closure.
```

Comment:
How does this interact with existing forks that had the right to fork previously?
Was it verified that this restriction can be applied on GitHub?

Reply:
The forks would be in violation, and step one would be to politely inform them that they need to remove the fork.

It was verified, but this is complicated and involves my agreement with GitHub, as well as the forking user's agreement with GitHub. I'll ask for that to be specifically reviewed.